react-graphql-apollo-tutorial.md

```diff
@@ -2611,15 +2607,15 @@ const RepositoryList = ({
 export default RepositoryList;
 {{< /highlight >}}
 
-In the *src/Issue/index.js* file, you should import and export the Issues component. Since the issue feature can be kept in a module on its own, it has this *index.js* file again. That's how you can tell other developers to only access this feature module by using the *index.js* file as its interface. Everything else is kept private in the feature module.
+In the *src/Issue/index.js* file, import and export the Issues component. Since the issue feature can be kept in a module on its own, it has this *index.js* file again. That's how you can tell other developers to access only this feature module, using the *index.js* file as its interface. Everything else is kept private.
 
 {{< highlight javascript >}}
 import Issues from './IssueList';
 
 export default Issues;
 {{< /highlight >}}
 
-Moreover, note how the component is only named Issues and not IssueList. Personally I like to make this naming convention to break down the rendering of a list of items: Issues, IssueList and IssueItem. Whereas Issues is the container component where you query the data and filter the issues eventually, and the IssueList and IssueItem are only there as presentational components for rendering purposes. In contrast, the Repository feature module hasn't such a Repositories component, because there was no need for it in the first place. The list of repositories already came from the Organization and Profile components and the Repository module's components are mainly only there for the rendering.
+Note how the component is named Issues, not IssueList. The naming convention is used to break down the rendering of a list of items: Issues, IssueList and IssueItem. Issues is the container component, where you query the data and filter the issues, and the IssueList and IssueItem are only there as presentational components for rendering. In contrast, the Repository feature module hasn't a Repositories component, because there was no need for it. The list of repositories already came from the Organization and Profile components and the Repository module's components are mainly only there for the rendering.
```

Comment:
Add in the end: However, keep in mind that this is only one opinionated approach of naming the components.